index.html

```diff
@@ -1734,28 +1781,77 @@ <h3>Document Encoding</h3>
+          conformance to profiles that the <a>transformation processor</a> does not support.</p>
+        <p>
+          After attribute value computation,
+          a <a>presentation processor</a> SHOULD ignore elements and attributes that it does not recognise and support.
+        </p>
+        <p class="note">
+          The above constraint is specified as being after attribute value computation because it is possible
+          that an implementation recognises and supports attributes present only on particular elements,
+          for example those corresponding to the DAPT data model.
+          As described in <a href="#using-computed-attribute-values"></a> it is important that processor implementations
+          do not ignore such attributes when present on other elements.
         </p>
-        <p class="ednote">Do we need to say that a <a>presentation processor</a> may ignore foreign vocab?</p>
 
         <section>
-          <h4>Proprietary Metadata</h4>
+          <h4>Proprietary Metadata and Foreign Vocabulary</h4>
           <p>Many dubbing and <a>audio description</a> workflows permit annotation of <a>Script Events</a> or documents with proprietary metadata.
             Metadata vocabulary defined in this specification or in [[TTML2]] MAY be included.
-            Additional vocabulary in other namespaces MAY also be included.</p>
+            <a>Foreign vocabulary</a> MAY also be included,
+            either as attributes of <code>&lt;metadata&gt;</code> elements or
+            as descendant elements of <code>&lt;metadata&gt;</code> elements.</p>
           <p class="note">It is possible to add information such as the title of the programme using [[TTML2]] constructs.</p>
+          <p>If any semantic needs to be defined for some <a>foreign vocabulary</a>,
+            or if it needs to be located outside a <code>&lt;metadata&gt;</code> element,
+            the mechanism provided to avoid it being pruned, and to indicate content and processor conformance, is as follows:</p>
+          <ol>
+            <li>Define a profile including a feature definition for that semantic and vocabulary,
+          with a profile designator.</li>
+            <li>Signal document conformance to that profile using the <a href="#ttp-contentprofiles"></a></li>
+          </ol>
+          <p>This allows processors that support the feature to process the vocabulary
+            in whatever way is appropriate, to avoid pruning it,
+            and allows processors that do not support the feature to take
+            appropriate action, for example warning users that some functionality may be lost.</p>
+          <p class="note">If <a>foreign vocabulary</a> is included in locations other than <code>&lt;metadata&gt;</code> elements
+            it will be pruned by <a>transformation processors</a> that do not support features associated with that vocabulary,
+            as required in <a href="#unrecognised-elements-and-attributes"></a>.</p>
```

Comment:
It's targeted at spec authors. I thought it would be helpful to explain how that use case has been thought about, and the tool provided. It could all be combined into a single note, I just thought it would be easier to read this way. Happy to adjust.

Reply:
Re-reading your note, I realise that you've mentioned forward/backward compatibility. That's not really what this text is about, it's more to do with foreign metadata. I'll refactor the text to make this bit more stand-alone and less intrusive within the simple section showing how to add other metadata.

Reply:
Hopefully this is better delineated and easier to skip past, for the uninterested, in f8c86e4.